website/Makefile

```diff
@@ -1,7 +1,47 @@
 deps:
-	rsync -av ../ content/ --include='assessments' --include='governance' --include='governance/related-groups' --include='supply-chain-security' --include='*.md' --exclude='*'
+	git submodule update --init --recursive # TODO: the hugo theme doesn't need to live in this directory
+
+	# Move select content from the root level into the website directory.
+	rsync -avv ../ root/ \
+	--include='assessments' --include='assessments/**' \
+	--include='governance' --include='governance/**' \
+	--include='supply-chain-security' --include='supply-chain-security/**' \
+	--include='*.md' --exclude='*'
+
+	# Move over content such as graphics and logos
 	rsync -av '../design/' 'static/design/' --exclude='#*'
-	git submodule update --init --recursive
+
+	# Update all imported markdown files to work as standalone hugo pages (except READMEs, see below)
```

Comment:
What do you think about checking for existing front-matter? Establishes an opt-in process for adjusting titles and other content.

Reply:
Seems like it would be a good improvement. Might be suited for a follow-up PR?

Reply:
Titles sometimes are duplicated - curious if we could check for existence and if not otherwise present - replace a `h1` with the same title?

Reply:
Good catch- glad you're doing a more thorough look through. I should have time on Friday to look for a fix unless you beat me to it.

Reply:
This is still an outlier - but maybe a feature we can add in a separate issue.